src/{% if use_ruff %}.ruff.toml{% endif%}.jinja

```diff
@@ -18,6 +18,9 @@ extend-select = [
     "{{ rule }}",
     {%- endfor %}
 ]
+{%- if odoo_version >= 15.0 %}
```

Comment:
could be as soon as 11.0 (python3)
although it's true that ruff is only enabled by default for repos >= 17.0

Reply:
yeah true, I updated it